--- a52dec-0.7.4/liba52/configure.incl.pic	2002-01-28 05:36:48.000000000 +0100
+++ a52dec-0.7.4/liba52/configure.incl	2003-10-02 15:33:31.000000000 +0200
@@ -2,7 +2,11 @@ AC_SUBST([LIBA52_CFLAGS])
 AC_SUBST([LIBA52_LIBS])
 
 dnl avoid -fPIC when possible
+dnl FIXME: fix libtool to not make it pass_all on AMD64?
+if [[ "$host_cpu" != "x86_64" ]]; then
 LIBA52_CFLAGS="$LIBA52_CFLAGS -prefer-non-pic"
+fi
+
 
 AC_ARG_ENABLE([double],
     [  --enable-double         use double-precision samples])
